src/main/java/uk/gov/hmcts/reform/prl/models/complextypes/QuarantineLegalDoc.java

```diff
@@ -80,6 +80,16 @@ public class QuarantineLegalDoc {
     private final Document cafcassQuarantineDocument;
     private final Document courtStaffQuarantineDocument;
 
+    // Adding Bulk scan attributes
+    public final String fileName;
```

Comment:
Same as asked in CCD PR, let's use `documentName` if possible instead of defining new file else new field is fine.